content/en/docs/setup/develop_with_kind.md

```diff
@@ -0,0 +1,124 @@
+---
+draft: false
+linktitle: Deploy/Develop with Kind
+menu:
+  docs:
+    parent: setup
+    weight: 4
+title: Deploy/Develop with Kind
+toc: true
+type: docs
+description: >
+  This guide provides an example for deploying and developing Kmesh.
+
+---
+# Kmesh Deploy and Develop in Kind
+
+[Kind](https://github.com/kubernetes-sigs/kind) is a convenient tool for quickly deploying kubernetes cluster locally. We can use `kind` to create an `istio` cluster and deploy `kmesh`.
```

Comment:
```suggestion
[Kind](https://github.com/kubernetes-sigs/kind) is a convenient tool for quickly deploying kubernetes cluster locally. We can use `kind` to create an `istio` cluster and deploy `Kmesh`.
```

Please use `Kmesh` uniformly, with the first letter capitalized.

Reply:
ok